.eslintrc.yaml

```diff
@@ -98,11 +98,18 @@ rules:
   func-call-spacing: 2
   func-name-matching: 2
   func-style: [2, declaration, {allowArrowFunctions: true}]
-  indent: [2, 2, {ArrayExpression: first,
-                  CallExpression: {arguments: first},
-                  MemberExpression: 1,
-                  ObjectExpression: first,
-                  SwitchCase: 1}]
+  # indent: [2, 2, {ArrayExpression: first,
```

Comment:
Do I understand it right that the plan is to land this PR, then fix the linter errors with the new `indent` rule in one or several follow-up PRs, and then uncomment this and remove the `indent-legacy` rule?